src/upper_envelope/fues_jax/fues_jax.py

```diff
@@ -182,6 +201,11 @@ def fast_upper_envelope(
     #     mask &= value < max_value_lower_bound
     #     value[mask] = jnp.nan
 
```

Comment:
Do we need the commented-out part still?

Reply:
Removed